caesar-cipher/caesar-cipher.c

```diff
@@ -0,0 +1,45 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+int main(){
+    int n; 
+    scanf("%d",&n);
```

Comment:
Please add print statements before taking Input & for Output
For Example:
```
Enter Input String Length: 3
Enter Input String: abc
Enter Key: 1
Output : bcd
```

Reply:
Fixed it.